cmd/geth/main.go

```diff
@@ -101,6 +101,7 @@ var (
 		utils.EnableProofKeeperFlag,
 		utils.KeepProofBlockSpanFlag,
 		utils.JournalFileFlag,
+		utils.EnableRecoverNodeBufferList,
```

Comment:
`EnableRecoverNodeBufferList ` -> `EnableRecoverNodeBufferListFlag ` to make a consistent style.

Reply:
Fixed